lib/omnifocus.rb

```diff
@@ -170,15 +171,15 @@ def create_missing_projects
   # for +bug_db+ for more info on how you should populate it.
 
   def update_tasks
-    bug_db.each do |name, tickets|
-      project = nerd_projects.projects[name]
+    bug_db.each do |project_name, tickets|
+      project = nerd_projects.projects[project_name]
```

Comment:
These diffs are unnecessary to _this_ patch. But are welcome clarification. Can you split that commit out to its own PR?